### ISSUE TITLE
Accessibility: pod status screen reader text missing on overview page

### ISSUE THREAD

**Issue by spadgett** [openshift/origin#8437]
The text simply says "Pod status:" with no details.

<img width="363" alt="screen shot 2016-04-09 at 8 26 11 am" src="https://cloud.githubusercontent.com/assets/1167259/14403606/e4d6f38e-fe2c-11e5-86f2-006b7a15bc3a.png">

/cc @jwforres

**Comment by spadgett** [comment]
This has since been fixed
